src/main/java/com/sap/oss/phosphor/fosstars/model/value/SecurityReviews.java

```diff
@@ -0,0 +1,143 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import static java.util.Arrays.asList;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * A set of security reviews.
+ */
+public class SecurityReviews implements Set<SecurityReview> {
+
+  /**
+   * Security reviews.
+   */
+  private final Set<SecurityReview> elements = new HashSet<>();
+
+  /**
+   * Create an empty set of security reviews.
+   */
+  public SecurityReviews() {
+
```

Comment:
Does not create any empty set.

Reply:
I am not sure that I got this statement (or question). This constructor creates an empty set. Jackson also calls it while deserialization.